src/core/object/watch/spec.js

```diff
@@ -851,6 +851,27 @@ describe('core/object/watch', () => {
 				expect(spy).not.toHaveBeenCalled();
 			});
 
+			it('setting up an object unwatchable', () => {
```

Comment:
Тест перенести после теста mute

```suggestion
			it('marking a part of the watched object as unwatchable', () => {
```